src/Support/Support.Autorest/docs/Get-AzSupportCommunication.md

```diff
@@ -40,27 +40,165 @@ Returns communication details for a support ticket.
+                                {font-family:Calibri}
+                         @font-face
+                                {font-family:Aptos}
+                         @font-face
+                                {font-family:"Segoe UI"}
+                         p.MsoNormal, li.MsoNormal, div.MsoNormal
+                                {margin:0in;
+                                font-size:12.0pt;
+                                font-family:"Aptos",sans-serif}
+                         a:link, span.MsoHyperlink
+                                {color:blue;
+                                text-decoration:underline}
+                         span.EmailStyle19
+                                {font-family:"Aptos",sans-serif;
+                                color:windowtext}
+                         .MsoChpDefault
+                                {font-size:11.0pt}
+                         @page WordSection1
+                                {margin:1.0in 1.0in 1.0in 1.0in}
+                         div.WordSection1
+                                {}
+                         -->
+                         </style></head><body lang="EN-US" link="blue" vlink="purple"
+                         style="word-wrap:break-word"><div class="WordSection1"><p class="MsoNormal"><span
+                         style="font-size:11.0pt">You can close these cases, thanks.</span></p><p
+                         class="MsoNormal"><span style="font-size:11.0pt">&nbsp;</span></p><p
+                         class="MsoNormal"><b><span style="font-size:10.0pt; font-family:&quot;Segoe
+                         UI&quot;,sans-serif; color:#505050">Jonathan Najjar <br></span></b><span
+                         style="font-size:10.0pt; font-family:&quot;Segoe UI&quot;,sans-serif; color:#505050">Software
+                         Engineer <br>Azure CXP <br><a href="mailto:jnajjar@microsoft.com"><span
+                         style="color:#467886">jnajjar@microsoft.com</span></a> <br></span><br><img border="0"
+                         width="115" height="28" id="Picture_x0020_1" src="cid:image001.png@01DA5F27.901D3F10"
+                         data-attachment-id="4ccf9f08-e323-48e6-87a7-8d5d77f7a1fd" alt="Microsoft Logo"
+                         style="width:1.1979in; height:.2916in"></p><p class="MsoNormal"><span
+                         style="font-size:11.0pt">&nbsp;</span></p><div style="border:none; border-top:solid #E1E1E1
+                         1.0pt; padding:3.0pt 0in 0in 0in"><p class="MsoNormal"><b><span style="font-size:11.0pt;
+                         font-family:&quot;Calibri&quot;,sans-serif">From:</span></b><span style="font-size:11.0pt;
+                         font-family:&quot;Calibri&quot;,sans-serif"> Melissa B
+                         &lt;support@mail.support.microsoft.com&gt; <br><b>Sent:</b> Tuesday, February 13, 2024 11:43
+                         AM<br><b>To:</b> Jonathan Najjar &lt;jnajjar@microsoft.com&gt;; Microsoft Support
```

Comment:
nit: could we replace the emails with dummy ones? Not sure if we should expose our colleague's real emails in the documentation

Reply:
added dummy names and email addresses